extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -1455,6 +1456,21 @@ export class AmpStoryEmbeddedComponent {
     );
   }
 
+  /**
+   * Linkers don't work on shadow root elements so we need to create an anchor outside it.
+   * @param {!Event} event
+   * @private
+   */
+  onAnchorClick_(event) {
+    event.preventDefault();
+    const outerAnchor = this.win_.document.createElement('a');
+    outerAnchor.href = this.tooltip_.href;
+    outerAnchor.target = this.tooltip_.target;
```

Comment:
There are a bit more parameters we need to support for analytics, can we clone the `a` node using `cloneNode` instead?

Can you check or upload a demo to verify this against many OS/browsers?

Reply:
Sounds good, I'll use `cloneNode` and set up a demo. The programmatic click works on Chrome/Mac but we can test it on other platforms as well